js/app.js

```diff
@@ -1,14 +1,7 @@
-$(document).ready(gameModule.init);
-
 var gameModule = {
-    init: func() {
+    init: function () {
         $(".back-panel").hide();
-        $(".game-name").off("click").on("click", function () {
-            $(".intro-panel").hide();
```

Comment:
store .intro-panel and .back-panel in variables, since they are used in multiple places